FirstPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,169 @@
+import Foundation
+
+
+// Функция решения квадратичного уравнения
+func getQuadEquationRoots(a: Double, b: Double, c: Double) -> Array<Double> {
+    var roots: Array<Double> = []
+    let d: Double = pow(b, 2) - 4 * a * c
+
+    if (d > 0) {
+        let squareRootD = sqrt(d)
+    
+        let root1: Double = (-b + squareRootD) / (2 * a)
+        let root2: Double = (-b - squareRootD) / (2 * a)
+
+        roots.append(root1)
+        roots.append(root2)
```

Comment:
Весь блок if {} можно заменить одной строкой
```suggestion
         return [ (-b + squareRootD) / (2 * a), (-b - squareRootD) / (2 * a) ]
```